frontend/src/pages/about/PagePublications.vue

```diff
@@ -6,7 +6,13 @@
 
 <template>
   <AppSection>
-    <AppHeading>Publications</AppHeading>
+    <AppHeading>Monarch-Related Publications</AppHeading>
+
+    <p class="metadata">
```

Comment:
Vestigial class?

Reply:
possibly, i added it to force the text to center and was too lazy at the time to figure out a better way to do that, but it worked. 
I'll change it out for whatever else you recommend though!

Reply:
If you're trying to center (as opposed to justify), then you can leave it. However other pages, e.g. the top of /teams, just leave it as justified.